include/tvm/relay/expr.h

```diff
@@ -248,6 +248,13 @@ class FunctionNode : public ExprNode {
    */
   TVM_DLL FuncType func_type_annotation() const;
 
+  /*!
+   * \brief Check whether the function is a primitive function.
+   *
+   * \return Whether the function is primitive or not.
+   */
+  bool IsPrimitive() const;
```

Comment:
Maybe replace this line with the new API?
https://github.com/dmlc/tvm/blob/01cfdb7ebcef0abc9282f68c3c4484974a6cc6e1/src/relay/pass/fuse_ops.cc#L702

Reply:
Yeah I was cherry-picking changes from an old branch 😄 got distracted at TVM conference while working on this PR.